### PR TITLE
Build python tools packager and fpgaflash through distutils

### PR DESCRIPTION
**Pull request by nakulkorde**

### CODE REVIEW

**Comment by nakulkorde** [comment]
The large number of files changes is due to the removal of jsonschema source from the repo.

File: tools/extra/packager/setup.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+setup(
+    name="opae.packager",
+    version="@INTEL_FPGA_API_VERSION@",
+    packages=find_packages(),
+    package_data={
+        'opae.tools.packager' : ['schema/*.json'],
+    },
+    entry_points={
+        'console_scripts': ['packager = opae.tools.packager:main'],
```
[the remark this answers is not in the document]
Comment (nakulkorde) [reply tools/extra/packager/setup.py]: Yes, if we can want it to be an entry point as well.

File: tools/extra/packager/setup.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    },
+    install_requires=['jsonschema'],
```
[the remark this answers is not in the document]
Comment (nakulkorde) [reply tools/extra/packager/setup.py]: Should the version be the same as before?

File: tools/extra/packager/setup.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (r-rojo) [reply tools/extra/packager/setup.py]: That script was previously being installed in bin and was callable so I think we want to keep it that way.

[the remark this answers is not in the document]
Comment (nakulkorde) [reply tools/extra/packager/setup.py]: Yes makes sense. I will add a fix.

File: tools/extra/packager/setup.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+setup(
+    name="opae.packager",
+    version="@INTEL_FPGA_API_VERSION@",
+    packages=find_packages(),
+    package_data={
+        'opae.tools.packager' : ['schema/*.json'],
+    },
+    entry_points={
+        'console_scripts': ['packager = opae.tools.packager:main',
+							'afu_json_mgr = opae.tools.packager.afu_json_mgr:main',
```
Comment (r-rojo) [inline tools/extra/packager/setup.py]: spacing is off

File: tools/extra/pac/fpgaflash/setup.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import os
```
Comment (r-rojo) [inline tools/extra/pac/fpgaflash/setup.py]: Missing copyright header

File: tools/extra/packager/setup.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import os
```
Comment (r-rojo) [inline tools/extra/packager/setup.py]: Missing copyright header

**Pull request merged**